django/http/response.py

```diff
@@ -50,7 +50,13 @@ def __init__(self, content_type=None, status=None, reason=None, charset=None):
         self.cookies = SimpleCookie()
         self.closed = False
         if status is not None:
-            self.status_code = status
+            try:
+                self.status_code = int(status)
+            except:
```

Comment:
Should I also include TypeError? An object would cause that

```
>>> int(object())
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: int() argument must be a string or a number, not 'object'
```